src/script/index.js

```diff
@@ -42,11 +42,38 @@ function getMolfile() {
 		{ ignoreErrors: true });
 }
 
+
+function getRepresentationInFormat(format) {
+	const input_mol = molfile.stringify(ketcher.editor.struct(),
+		{ ignoreErrors: true });
+	return new Promise((resolve) => {
+		$.ajax({
+			type: "POST",
+			url: '/plexus/rest-v0/util/calculate/stringMolExport',
+			data: { structure: input_mol, parameters: format }
+		}).done(function updateInputCompound(smiles) {
+			resolve(smiles);
+		});
+	});
+}
+
+function getSvg() {
+	const mol = Module.get_mol(molfile.stringify(ketcher.editor.struct(), // eslint-disable-line no-undef
+		{ ignoreErrors: true }));
+	return mol.get_svg();
+}
+
 function setMolecule(molString) {
 	if (!(typeof molString === 'string'))
 		return;
-	ketcher.ui.load(molString, {
-		rescale: true
+	$.ajax({
+		type: "POST",
+		url: '/plexus/rest-v0/util/calculate/stringMolExport',
+		data: { structure: molString, parameters: 'MOL' }
+	}).done(function updateInputCompound(smiles) {
```

Comment:
should we also a minimal `.fail` which just console logs the failure?

Reply:
Done